src/lib/y2storage/proposal/planned_devices_generator.rb

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2016] SUSE LLC
```

Comment:
This is a renamed file, not really a new one. The original is from 2016 and we usually don't update copyright note. Of course, I can do it for this one.